kitsune/search/v2/documents.py

```diff
@@ -129,7 +137,7 @@ class Index:
         using = config.DEFAULT_ES7_CONNECTION
 
     def prepare_question_tag_id(self, instance):
```

Comment:
I'd also rename the field to `question_tag_ids` to be consistent with our other naming